src/OrchardCore.Modules/OrchardCore.Email.Azure/Manifest.cs

```diff
@@ -1,14 +1,14 @@
 using OrchardCore.Modules.Manifest;
 
 [assembly: Module(
-    Name = "Azure Email Provider",
+    Name = "Azure Communication Services (Email)",
```

Comment:
Why not "Azure Communication Services Email" and "Azure Communication Services SMS"? Since it's an important part that these are email and SMS providers (parenthesis are used for something secondary).

Reply:
Because as you pointed out the service is called "Azure Communication Services" but the channel is Email or SMS. So I used the service name and highlighting the channel `(Email)` or `(SMS)`

Reply:
Parenthesis don't highlight, though, they de-emphasize.

Reply:
okay. I changed it.